test/Garnet.test/RespSetTest.cs

```diff
@@ -50,6 +50,27 @@ public void CanAddAndListMembers()
             Assert.AreEqual(expectedResponse, actualValue);
         }
 
+        [Test]
+        public void CanCheckIfMemberExistsInSet()
```

Comment:
Please add a test for LightClient as well (for reference see other tests ending in 'LC')

Reply:
done